lib/src/feedback_widget.dart

```diff
@@ -204,6 +207,7 @@ class FeedbackWidgetState extends State<FeedbackWidget>
     ScreenshotController controller,
     String feedbackText, {
     Duration delay = const Duration(milliseconds: 200),
+    double pixelRatio = 3.0,
```

Comment:
Same here

Reply:
You mean you want this mandatory on the function calls as well?